components/FavCard.js

```diff
@@ -1,10 +1,13 @@
 import React, { useState, useCallback } from 'react';
-import {StyleSheet, View, FlatList} from 'react-native';
+import {StyleSheet, View, FlatList, Dimensions} from 'react-native';
 import {BoxShadow} from 'react-native-shadow';
 import COLORS from '../global-styles/Colors.js';
-import RecText from '../components/RecText.js'
-import {typeStyles} from '../global-styles/typeStyles.js'
-import LeagueTopper from '../components/LeagueTopper.js'
+import LeagueTopper from '../components/LeagueTopper.js';
+import Matchup from '../components/Matchup';
+import FavCardTabView from './FavCardTabView.js';
+
+const width = Dimensions.get('window').width;
+const cardWidth = width-32;
 
 const useComponentSize = () => {
     const [size,setSize] = useState({height: 360});
```

Comment:
nothing to change now but we may want to consider making this function importable if its going to be used a lot in other places